shell/common/platform_view.cc

```diff
@@ -85,6 +85,13 @@ void PlatformView::UpdateSemantics(
     blink::SemanticsNodeUpdates update,
     blink::CustomAccessibilityActionUpdates actions) {}
 
+bool PlatformView::RegisterSemanticsAvailableCallback(fml::closure closure) {
+  FML_LOG(WARNING) << "The default implementation of "
```

Comment:
Just a thought, but it might not be clear to the developer what alternative there is to the "default implementation".  I know for me, despite lots of tracing through the C++ code, I'm still surprised by many delegates and I still don't have a meaningful appreciation for why certain things are implemented downstream.  It might be helpful to point the developer in the direction of where they might find the desired implementation.

Reply:
In this case, it'd be up to the embedder to implement it.  If you're consuming this in an add2app scenario, I want to make sure it's clear if your embedding treats it as a no-op.  Maybe this message would be better as  `"The embedder for your platform does not implemented this feature, and ..."`?

Reply:
That's probably a step in the right direction.  Anything that helps suggest what a developer might need to do, or where to look, or what to submit in a bug report, I think is preferable to a mere statement of fact that may or may not hold significance to the reader.